internal/metrics/directory_usage_collector.go

```diff
@@ -0,0 +1,60 @@
+package metrics
+
+import (
+	"github.com/alecthomas/units"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	"golang.org/x/sys/unix"
+)
+
+const (
+	directoryUsageGbName                = "assisted_installer_directory_usage_gb"
+	directoryUsageGbDescription         = "The amount of the free space in a directory that has been used in Gb"
```

Comment:
Wouldn't be best to export it in bytes, and let the UI layer do the transformation to whatever unit it's most comfortable?